src/data/routing/Routing.js

```diff
@@ -0,0 +1,46 @@
+/*
+
+@params
+Input
+data: Object
+
+Output
+percentage: string
+
+@description
+Calculate the usage % of driver.
+
+*/
+
+export var calculateDriverUsage = function(data) {
```

Comment:
- Please use an arrow function here:
```js
export const calculateDriverUsage = (data) => {...}
```
- Never use `var` it is not block scoped